metadata/src/test/java/org/apache/kafka/metalog/LocalLogManagerTestEnv.java

```diff
@@ -87,11 +89,20 @@ public Builder setSharedLogDataInitializer(Consumer<SharedLogData> sharedLogData
             return this;
         }
 
+        /**
+         * Used to mock the latest KRaft version that would be returned from RaftClient.kraftVersion()
+         */
+        public Builder setLastKRaftVersion(short kraftVersion) {
```

Comment:
I would change the signature to `setLastKRaftVersion(KRaftVersion` and make the test do the conversion from `short` to `KRaftVersion`.